src/app/InteractionModelEngine.cpp

```diff
@@ -1060,16 +1060,23 @@ bool InteractionModelEngine::EnsureResourceForSubscription(FabricIndex aFabricIn
 
     countResourceUsage();
 
+    ChipLogDetail(InteractionModel, "Fabric [%d] subscription resources: attribute paths %d + %d <= %d, event paths %d + %d <= %d, read handlers %d + 1 <= %d",
+        aFabricIndex, 
+        usedAttributePaths, aRequestedAttributePathCount, attributePathCap,
+        usedEventPaths, aRequestedEventPathCount, eventPathCap,
+        usedReadHandlers, readHandlerCap);
     if (usedAttributePaths + aRequestedAttributePathCount <= attributePathCap &&
-        usedEventPaths + aRequestedEventPathCount <= eventPathCap && usedReadHandlers < readHandlerCap)
+        usedEventPaths + aRequestedEventPathCount <= eventPathCap && 
```

Comment:
Why bother changing with cosmetic changes like this in code that's not maintained by us? 
I think we should keep our deviations to a minimum to reduce the number of potential conflicts in case we need to upgrade to a later version of connectedhomeip.

Reply:
True, I just did it because I found it hard to read while trying to find the root cause. I'll try to do it less in the future. For now merging